doc/design/cpp_data_feeding.md

```diff
@@ -69,10 +113,59 @@ To solve this problem, we introduce `ReaderHolder` as a wrapper. It acts as an e
 
 To create and invoke readers, some new ops are introduced:
 
-### `CreateReaderOp`
+### CreateReaderOp
```

Comment:
Why do we need `CreateReaderOp` if there are creator operator for each kind of reader?

Reply:
`CreateReaderOp` is just a general name for all reader creator operators. There is no such `CreateReaderOp`.

Reply:
Maybe rename it to "### Operators That Creates Readers"?